internal/database/database_container.go

```
@@ -360,29 +479,16 @@ func (i *Instance) Handle() (result *sql.DB, err error) {
 	return
 }
 
-// Close deletes the database and the user that were created for this instance.
+// Close deletes the database that was created for this instance.
 func (i *Instance) Close() error {
-	var errs []error
-
-	// Drop the database:
-	_, err := i.server.handle.Exec(fmt.Sprintf(
+	_, err := i.container.handle.Exec(fmt.Sprintf(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)